2-mandatory/2-financial-times.js

```diff
@@ -14,7 +22,19 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+  const titleLength = [];
+
+  for (let title of allArticleTitles) {
+    titleLength.push(title.length);
+  }
+
+  let miniTitle = Math.min(...titleLength);
+
+  for (let title of allArticleTitles)
+    if (title.length == miniTitle) {
+      return title;
+    }
```

Comment:
Well done @AppolinFotso. Very cool and clean method to find shortest title. I also just learned of the triple dot operator from your code, looks very useful thank you!

Reply:
Thank you @ManSangSin! I also learned a lot from your codes.